source/opt/ir_loader.cpp

```diff
@@ -99,7 +99,9 @@ bool IrLoader::AddInstruction(const spv_parsed_instruction_t* inst) {
     dbg_line_info_.clear();
   } else if (last_line_inst_ != nullptr) {
     last_line_inst_->SetDebugScope(last_dbg_scope_);
-    spv_inst->dbg_line_insts().push_back(*last_line_inst_);
+    if (extra_line_tracking_) {
```

Comment:
Minor issues.

We do not need the line 95 to the line 98 if `extra_line_tracking_` is false.
The following code will let use skip cloning `last_line_inst_`:
```
    if (extra_line_tracking_ && spv_inst->dbg_line_insts().back().opcode() != SpvOpNoLine) {
      last_line_inst_ = std::unique_ptr<Instruction>(
          spv_inst->dbg_line_insts().back().Clone(module()->context()));
    }
```
Since the above change will make `last_line_inst_` always `null` if `extra_line_tracking_` is false, we do not need additional if statement in the line 102.

Reply:
Done.  I also updated the comment on last_line_inst_ in the header because now it's always null when extra_line_tracking_ is false.